cdn_lambda/functions/origin_response/origin_response.py

```diff
@@ -0,0 +1,29 @@
+import logging
+import re
+
+LOG = logging.getLogger("origin-response")
+
+
+def lambda_handler(event, context):
+    # pylint: disable=unused-argument
+
+    request = event["Records"][0]["cf"]["request"]
+    original_uri = request["headers"]["original-uri"][0]["value"]
+
+    response = event["Records"][0]["cf"]["response"]
+
+    max_age_pattern_whitelist = [
+        ".+/PULP_MANIFEST",
+        ".+/listing",
+        ".+/repodata/repomd.xml",
+        ".+/ostree/repo/refs/heads/.+/.+",
+    ]
+
+    for pattern in max_age_pattern_whitelist:
+        if re.search(pattern, original_uri):
+            response["headers"]["cache-control"] = [
+                {"key": "Cache-Control", "value": "max-age=600"}
```

Comment:
I think the issue description asked for the number to be configurable.  Could that please be arranged?  e.g. the number loaded from the config file.

If so, the config file probably should no longer be named map_to_s3.json too.